plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/TrinoS3FileSystem.java

```diff
@@ -1487,6 +1489,13 @@ public void close()
             }
         }
 
+        @Override
+        public long getRetainedSizeInBytes()
+        {
+            // TODO find a better way to estimate this
```

Comment:
If `transferManager` does not hold for memory outside between `TrinoS3StagingOutputStream` calls, then it should be 0

Reply:
I was not sure about this but you are right probably